skimage/io/collection.py

```diff
@@ -139,9 +139,9 @@ def imread_convert(f):
     Examples
     --------
     >>> import skimage.io as io
-    >>> from skimage import data_dir
+    >>> from skimage.data import fetch
 
-    >>> coll = io.ImageCollection(data_dir + '/chess*.png')
+    >>> coll = io.ImageCollection([fetch('chessboard_GRAY.png'), fetch('coins.png')])
```

Comment:
I feel like this is a problem, and maybe the hacky class is needed, if pooch can't handle glob expansions. Which would make sense.